cmd/jaeger/internal/all-in-one.yaml

```diff
@@ -30,8 +30,9 @@ extensions:
   remote_sampling:
     # We can either use file or adaptive sampling strategy in remote_sampling
     file:
-      path: ./cmd/jaeger/sampling-strategies.json
+      path:
       reload_interval: 1s
+      default_sampling_probability: 0.001
```

Comment:
why not 1.0? I think this would be equivalent to the previous config where the stategies file was specified